src/components/Room.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled, { createGlobalStyle } from "styled-components";
+
+export const StyledMain = styled.main`
+	background: #c7ddc9;
+	height: 93vh;
+	padding: 1%;
```

Comment:
```suggestion
```

I'm not sure if this is needed.

When/If you move the `createGlobalStyle` to a component in `main.tsx`, you probably want to add the padding to all the pages (LandingPage, DefaultPage and Room), so adding it here would not be the best place.

You can use your judgment to see where this padding fits in best (I'm not very sure)